### PR TITLE
Suggested corrections by chapter

### PR DESCRIPTION
**Pull request by elliottinvent**

### CODE REVIEW

**Comment by elliottinvent** [comment]
Earlier I could see based on the Netlify PR deploy preview that the fixes I suggested for rendering "_hyperscript" in Asciidoc (`//_hyperscript`) hadn't worked, so I pushed a new commit to use a single backslash but that didn't work either.

I can't see any rhyme or reason as to why some instances of "_hyperscript" with a single backslash show fine and others don't – maybe it's a problem with the library you're using to render the Asciidoc to HTML? 

In my most recent commit I've used `+_hyperscript+` and this [seems to be working](https://deploy-preview-15--hypermedia-systems.netlify.app/) although my Sublime Text syntax helper doesn't recognise it. 

To me at least, the [Asciidoc guidance](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/) on when to use `+` or `++` is as clear as mud, but this seems to be working!

File: book/CH07_MorehtmxPatterns.adoc [hunk 1]
```diff
@@ -597,12 +597,12 @@ Beautiful.
...
 Of course, as you probably suspected, all is not perfect.  Unfortunately, upon shipping this feature to production, we
 start getting some complaints from the users that the application "`feels slow`".  Like all good developers faced with
-a performance issues, rather than guessing what the issue might be, we try to get a performance profile of the application
+a performance issue, rather than guessing what the issue might be, we try to get a performance profile of the application
 to see what exactly is causing the problem.
 
 It turns out, surprisingly, that the problem is that innocent looking `Contacts.count()` call, which is taking up to
 a second and a half to complete.  Unfortunately, for reasons beyond the scope of this book, it is not possible to improve
-that load time, nor it is also not possible to cache the result.
+that load time, nor is possible to cache the result.
```
Comment (1cg) [inline book/CH07_MorehtmxPatterns.adoc:605]: "is it" probably?

**Review by 1cg — CHANGES_REQUESTED** [review]
holy smokes what an editing pass!  thank you!

i found one small edit to an edit, but otherwise looks great!

File: book/CH07_MorehtmxPatterns.adoc (hunk 1 above)
Comment (elliottinvent) [reply book/CH07_MorehtmxPatterns.adoc:605]: Yes, missed that one. Thanks!

**Review by 1cg — APPROVED** [review]

**Comment by 1cg** [comment]
merged, thank you!

**Pull request merged**